src/GraphQl/Resolver/Stage/DeserializeStage.php

```diff
@@ -54,6 +54,12 @@ public function __invoke($objectToPopulate, string $resourceClass, string $opera
             $denormalizationContext[AbstractNormalizer::OBJECT_TO_POPULATE] = $objectToPopulate;
         }
 
-        return $this->denormalizer->denormalize($context['args']['input'], $resourceClass, ItemNormalizer::FORMAT, $denormalizationContext);
+        $item = $this->denormalizer->denormalize($context['args']['input'], $resourceClass, ItemNormalizer::FORMAT, $denormalizationContext);
+
```

Comment:
Remove blank line?

Reply:
A matter of taste, doesn't it?

Reply:
True... Haha. But looking elsewhere in our code, this seems to be the style.